bedrock/settings/static_media.py

```diff
@@ -1013,6 +1013,18 @@
         ),
         'output_filename': 'js/common-bundle.js',
     },
+    'common-oldie': {
+        'source_filenames': (
+            'js/libs/jquery-1.11.3.min.js',  # upgrade to jQuery 1.12?
+            # will need to make sure scripts below are compatible with jQuery 1.12 & 3?
```

Comment:
> Well...maybe. If we want to hide the footer lang chooser from IE 8 and below (or just accept that it wont work), we can get rid of global.js and global-init.js. I'd be fine hiding the lang chooser from old browsers. Thoughts?

The footer lang switcher works without JS, but looks like it relies on `<noscript>`. Could we change this to use the `no-js` class instead?